bootstrap-app/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:7
+
+# Install Java and clean up.
+RUN curl -LO --cookie 'oraclelicense=accept-securebackup-cookie' 'http://download.oracle.com/otn-pub/java/jdk/8u45-b14/jdk-8u45-linux-x64.rpm' \
+ && rpm -Uvh jdk-8u45-linux-x64.rpm \
+ && rm jdk-8u45-linux-x64.rpm
+ENV JAVA_HOME /usr/java/jdk1.8.0_45/jre
+
+COPY . target/cmr-bootstrap-app-0.1.0-SNAPSHOT-standalone.jar
+
```

Comment:
Only access-control was updated to add the port. We should update all of them to expose the default port. You can get the port numbers in `cmr.transmit.config`